recipe/meta.yaml

```diff
@@ -52,19 +52,19 @@ requirements:
     - packaging >=20.0
     - typing_extensions {{ typing_extensions }}  # [py<38] 
     - jinja2
+    - langcodes
```

Comment:
Do we not need any pin for langcodes?

Reply:
langcodes is used only by spacy and is built by us, so not added pin for it in spacy recipe